docs/sites/breaking.rst

```diff
@@ -0,0 +1,10 @@
+Breaking changes
+================
+
+Breaking changes to existing APIs are naturally disruptive to users, but sometimes
+necessary in order to introduce new functionality.
+When we intend to make breaking changes to user-facing APIs in dataframely, we aim to make it easy for users to anticipate such changes and migrate their code to the new behavior.
+
+``FutureWarnings``
+----------------
```

Comment:
Does this render correctly?

```suggestion
``FutureWarnings``
------------------
```

Reply:
I think thats more of a guideline that a rule, it seems to render correctly either way (on my machine :tm:), anyway accepted the suggestoin